testMVVM/EpisodeCreateCoordinator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  EpisodeCreateCoordinator.swift
+//  MVVMTestProject
+//
+//  Created by Petr Zvoníček on 25.09.16.
+//  Copyright © 2016 Martin Pinka. All rights reserved.
+//
+
+import UIKit
+
+class EpisodeCreateCoordinator: Coordinator {
+
+    let navigationController: UINavigationController
+    let viewModel: SeasonDetailViewModel
+
+    let wrapperNavigationController: UINavigationController?
+    var viewController: EpisodeCreateViewController
+
+    init(navigationController: UINavigationController, wrapperNavigationController: UINavigationController, viewController: EpisodeCreateViewController, viewModel: SeasonDetailViewModel) {
```

Comment:
the parameter wrapperNavigationController is required but property of coordinator is optional. 
In the start method you check if wrapperNavigationController is available but it is always available because of init method